src/Symfony/Component/Form/Extension/Core/Type/FormType.php

```diff
@@ -177,10 +178,12 @@ public function configureOptions(OptionsResolver $resolver)
             'attr' => array(),
             'post_max_size_message' => 'The uploaded file was too large. Please try to upload a smaller file.',
             'upload_max_size_message' => $uploadMaxSizeMessage, // internal
+            'help' => '',
```

Comment:
Should the default be an empty string or null? Spontaneously I would say `null`, but Im sure there is a reason you chose the empty string.

Reply:
I've not test with null... But In the line bellow, we must define the accepted type that is string. That's why I've put an empty string. But Maybe it works well with null, I can try it. Tests will scream if it's not good.

Reply:
I confirm, if default on null, it return error because is expect a string value.

Reply:
Okey, Could we make it expect a string value and null? Or maybe `false` would make more sense.

Reply:
I can try if I can give him an array of possibilities, but I don't know if it works, I don't know this part of the code.

Reply:
It's okay :-)

    $resolver->setAllowedTypes('help', ['string', 'NULL']);

Reply:
Great